packages/date-time/src/components/Calendar/examples/Calendar.Inline.ContiguousWorkWeekDays.Example.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Calendar, DateRangeType, DayOfWeek, defaultDayPickerStrings } from '@uifabric/date-time';
+
+import * as styles from './Calendar.Example.scss';
+
+export interface ICalendarInlineExampleState {
+  selectedDate?: Date;
+  selectedDateRange?: Date[];
+}
+
+export class CalendarInlineContiguousWorkWeekDaysExample extends React.Component<{}, ICalendarInlineExampleState> {
```

Comment:
Is this just a file move, or a new example? It would be good to make this a simple function component. Could happen separately in a PR.

Reply:
I'd like to get these checked in sooner rather than later, but will make a note to convert them to functional components in a future PR. This is basically just breaking apart the old example (which accepted props) into multiple example files that don't have props, to support storybook scenario. The conversion to functional components will require a little more time